Kconfig

```diff
@@ -928,6 +928,18 @@ menu "LVGL configuration"
 		config LV_USE_KEYBOARD
 			bool "Keyboard."
 			default y if !LV_CONF_MINIMAL
+		config LV_USE_KEYBOARD_PINYIN
```

Comment:
I though it's a new widget but from the code, it's an function extension to existing keyboard widget. How about calling is `LV_KEYBOARD_PINYIN`?

Reply:
> I though it's a new widget but from the code, it's an function extension to existing keyboard widget. How about calling is `LV_KEYBOARD_PINYIN`?

ok